features/webnn.yml

```diff
@@ -0,0 +1,3 @@
+name: WebNN
+description: The WebNN API constructs and executes computational graphs of neural networks by making use of the various machine learning capabilities and hardware accelerators available on the device. Also known as Web Neural Network API.
+spec: https://www.w3.org/TR/webnn/
```

Comment:
Can you add this to the allowed spec lists? Background in https://github.com/mdn/browser-compat-data/pull/22569#issuecomment-1992632118

In https://github.com/w3c/browser-specs/issues/1268 I made an attempt at automating this, but it's not tractable yet.